bundle.Dockerfile

```diff
@@ -5,7 +5,7 @@ LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=cluster-logging-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=4.6
-LABEL operators.operatorframework.io.bundle.channel.default.v1=
+LABEL operators.operatorframework.io.bundle.channel.default.v1=4.6
 
-COPY manifests /manifests/
-COPY metadata /metadata/
+COPY bundle/manifests /manifests/
```

Comment:
We can remove `bundle/manifest/kustomization.yaml` and `bundle/manifest/image-references` after the `COPY` step.
As these are not part of bundle, and cause validation failures.

Reply:
they are removed with the `find` command in `generate-bundle`